src/Text/Pretty/Simple/Internal/Expr.hs

```diff
@@ -43,5 +45,6 @@ data Expr
   -- only thing we will be doing with it is turning it /back/ into a string
   -- at some stage, so we might as well cut out the middle man and store it
   -- directly like this.
+  | CustomExpr Style !String
   | Other !String
-  deriving (Data, Eq, Generic, Show, Typeable)
```

Comment:
We could probably recover this by adding standalone-derived orphans for the `ansi-terminal` types.